spec.html

```diff
@@ -11892,7 +11892,7 @@
           1. Let _tailCall_ be IsInTailPosition(_thisCall_).
           1. Return ? EvaluateDirectCall(_func_, _thisValue_, |Arguments|, _tailCall_).
         </emu-alg>
-        <p>A |CallExpression| evaluation that executes step 3.a.vi is a <dfn>direct eval</dfn>.</p>
+        <p>A |CallExpression| evaluation that executes step 3.a.vii is a <dfn>direct eval</dfn>.</p>
```

Comment:
> I too wish there were a way to link to steps in a stable manner without requiring each step to be annotated with a unique id.

If, by "each step", you mean "each step that the spec actually references", then what's so bad about annotating each with a unique id? There's only about 50 of them.

(And if you meant "every step in the spec", then I'm not sure why you're thinking that would be necessary.)

> You could consider a hash of the step contents but then minor editorial changes would cause problems...

Moreover, step contents aren't unique, so there'd be the risk of hash collisions.